Stripe/PublicHeaders/STPPaymentIntentAction.h

```diff
@@ -0,0 +1,50 @@
+//
+//  STPPaymentIntentAction.h
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 3/8/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "STPAPIResponseDecodable.h"
+#import "STPPaymentIntentEnums.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class STPPaymentIntentActionRedirectToURL;
+
+/**
+ Source Action details for an STPPaymentIntent. This is a container for
```

Comment:
Should this be "PaymentMethod Action", or just "Action"?

Reply:
Changing to "Action"